RecoverVm/AttachOsDiskAsDataDiskToRecoveryVm.ps1

```diff
@@ -0,0 +1,63 @@
+﻿function AttachOsDiskAsDataDiskToRecoveryVm   (
+    [string] $ServiceName , 
+    [string] $VName ,    
+    [string] $RecoveryAdmin = 'recoveryAdmin',
+    [string] $RecoveryPW = [guid]::NewGuid().ToString()
+    )
+{
+    $RecoveryVMName = 'RC' + $(get-date -f yyMMddHHmm)    
+
+    #get defect vm object and export cfg to disk (temp)
+    $vm = Get-AzureVM -ServiceName $ServiceName -Name $VMName
+
+    if ( ! $vm ) 
+    {
+        return
```

Comment:
What is the expected experience if someone tried to point it to an ARM VM or has a typo in the VM or service name? It looks like we’d return here and then we’d error out dereference the empty return variable in the calling function.